config/php-cs-fixer.php

```diff
@@ -71,7 +71,9 @@
             'php_unit_mock_short_will_return' => true,
             'php_unit_set_up_tear_down_visibility' => true,
             'php_unit_test_annotation' => ['style' => 'annotation'],
-            'php_unit_test_case_static_method_calls' => ['call_type' => 'self'],
+            // `'methods' => ['createStub' => 'this']` is required as long as we are not on PHPUnit 10.x yet
+            // (where that method was made static).
+            'php_unit_test_case_static_method_calls' => ['call_type' => 'self', 'methods' => ['createStub' => 'this']],
```

Comment:
I'm surprised that we need this, given that we do not call `createStub` AFAICS, and neither does PHPUnit itself.  I'm also mystified as to what change between 3.64 and 3.66 of the Fixer means that we would now need it.